pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/LinuxInfoUtils.java

```diff
@@ -132,7 +133,7 @@ public static ResourceUsage getCpuUsageForEntireHost() {
                     .filter(s -> !s.contains("cpu"))
                     .mapToLong(Long::parseLong)
                     .sum();
-            long idle = Long.parseLong(words[4]);
+            long idle = Long.parseLong(words[4]) + Long.parseLong(words[5]);
```

Comment:
It will be a little dangerous to split values directly, can it be handled by oshi library?

Reply:
If we consider using `oshi` library, may be we should rewrite whole module using `oshi`?  
`oshi` support for  retrieving system information, such as memory and CPU usage, disks and partitions, etc.
To fix the `iowait` problem, a simple patch above may be a better solution. 
@mattisonchao

Reply:
The current implementation of module is splitting values directly.
```
long idle = Long.parseLong(words[4]);
```
there is no different with the patch.
```
long idle = Long.parseLong(words[4]) + Long.parseLong(words[5]);
```
I think we could fix this problem with this patch first. 
WDYT. @codelipenghui